main.js

```diff
@@ -1,15 +1,333 @@
+
+    const bioPop = document.createElement('p');
+    bioPop.classList.add('popup-info');
+    bioPop.appendChild(document.createTextNode(projectData.persona));
+    popupContent.appendChild(bioPop);
+
+    const languagesPop = document.createElement('ul');
+    languagesPop.classList.add('card-languages');
+
+    const htmlPop = document.createElement('li');
+    const cssPop = document.createElement('li');
+    const jsPop = document.createElement('li');
+
+    popupContent.appendChild(languagesPop);
+    languagesPop.appendChild(htmlPop);
+    languagesPop.appendChild(cssPop);
+    languagesPop.appendChild(jsPop);
+
+    htmlPop.appendChild(document.createTextNode(projectData.languages[0]));
+    cssPop.appendChild(document.createTextNode(projectData.languages[1]));
+    jsPop.appendChild(document.createTextNode(projectData.languages[2]));
+
+    // eslint-disable-next-line max-len
+    if (projectData.languages[3] != null && projectData.languages[4] != null && projectData.languages[5] != null) {
+      const githubPop = document.createElement('li');
+      const rubyPop = document.createElement('li');
+      const bootstrapPop = document.createElement('li');
+
+      languagesPop.appendChild(githubPop);
+      languagesPop.appendChild(rubyPop);
+      languagesPop.appendChild(bootstrapPop);
+
+      githubPop.appendChild(document.createTextNode(projectData.languages[3]));
+      rubyPop.appendChild(document.createTextNode(projectData.languages[4]));
+      bootstrapPop.appendChild(document.createTextNode(projectData.languages[5]));
+
+      githubPop.classList.add('card-lang1');
+      rubyPop.classList.add('card-lang1');
+      bootstrapPop.classList.add('card-lang3');
+    }
+
+    htmlPop.classList.add('card-lang1');
+    cssPop.classList.add('card-lang2');
+    jsPop.classList.add('card-lang3');
+
+    const hrPop = document.createElement('hr');
+    hrPop.classList.add('line');
+    popupContent.appendChild(hrPop);
+
+    const liveSource = document.createElement('div');
+    liveSource.className = 'live-source';
+    popupContent.appendChild(liveSource);
+
+    const liveButton = document.createElement('button');
+    liveButton.setAttribute('class', 'live');
+    liveButton.type = 'button';
+    liveSource.appendChild(liveButton);
+    liveButton.appendChild(document.createTextNode('See Live'));
+
+    const liveIcon = document.createElement('img');
+    liveIcon.setAttribute('class', 'liveImg');
+    liveIcon.src = './img/Icon-live.svg';
+    liveButton.appendChild(liveIcon);
+
+    document.querySelectorAll('.live').forEach((live) => {
+      live.addEventListener('click', () => {
+        window.open(projectData.live);
+      });
+    });
+
+    const sourceButton = document.createElement('button');
+    sourceButton.setAttribute('class', 'source');
+    sourceButton.type = 'button';
+    liveSource.appendChild(sourceButton);
+    sourceButton.appendChild(document.createTextNode('See Source '));

```

Comment:
- [ ] Kindly vertically align the text and icon in the button to the center for all modal popups, this will make your design look like the Figma design 👍🏿 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_446](https://user-images.githubusercontent.com/69638013/190384925-47d208e8-dacb-4e93-81f1-0821306ef329.png) | ![Screenshot_445](https://user-images.githubusercontent.com/69638013/190384798-7940308b-454b-4f82-93f5-f85cf3392769.png)